doc/optparse/ruby/basic.rb

```diff
@@ -12,5 +12,6 @@
 parser.on('-z', 'Whether to Z') do |value|
   p ['z', value]
 end
-# Parse the command line.
-parser.parse!
+# Parse the command line and return pared-down ARGV.
```

Comment:
Can we find a word which is more common than pared-down? Here, I'm thinking of readers whose not first-language English.

I see the specificity, though. Terse etc. 

(Oh, I'll get back to this, gotta sleep!)